README.md

```diff
@@ -0,0 +1,100 @@
+# HTML & CSS3 & JavaScript Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
+## Set-up GitHub Actions
+
+Please do the following **steps in this order**:
```

Comment:
- [ ] This readme doesn't describe your project. Please use this [readme template](https://github.com/microverseinc/readme-template) and ensure you customise it to your project.

Reply:
@yahyaelganyni1 
The first and second reviewers explicitly told you to work on your README file, in fact, you were given the [template](https://github.com/microverseinc/readme-template) up there 👆 .

KIndly follow the template and update your README file.